src/components/layouts/NavigationBar/NavigationBar.tsx

```diff
@@ -1,10 +1,16 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { FaUserCircle } from 'react-icons/fa';
 import { Link } from 'react-router-dom';
-
+import UserContext from '../../../hooks/UserContext';
 import './NavigationBar.scss';
 
 const NavigationBar = (): JSX.Element => {
```

Comment:
I would recommend using React.FC declaration:`const NavigationBar: React.FC<{}> = () => {...`
Here is an awesome React typescript style guide: https://github.com/ykeanu/react-typescript-style-guide